nav2_costmap_2d/plugins/static_layer.cpp

```diff
@@ -138,6 +138,7 @@ StaticLayer::getParameters()
   declareParameter("transform_tolerance", rclcpp::ParameterValue(0.0));
   declareParameter("map_topic", rclcpp::ParameterValue("map"));
   declareParameter("footprint_clearing_enabled", rclcpp::ParameterValue(false));
+  declareParameter("restore_outdated_map", rclcpp::ParameterValue(false));
```

Comment:
Lets rename this + the variable to something like `restore_cleared_footprint`. Make sure to also update the migration guide and configuration docs with this.

Lets default to true, I think this is the most reasonable behavior!